resources/rebuild.sh

```diff
@@ -244,48 +216,53 @@ function build_al_kernels {
 
     clone_amazon_linux_repo
 
-    # Apply kernel patches on top of AL configuration
-    apply_kernel_patches_for_ci
+    CI_CONFIG="$PWD/guest_configs/ci.config"
 
     if [[ "$KERNEL_VERSION" == @(all|5.10) ]]; then
-        build_al_kernel $PWD/guest_configs/microvm-kernel-ci-$ARCH-5.10.config
+        build_al_kernel $PWD/guest_configs/microvm-kernel-ci-$ARCH-5.10.config "$CI_CONFIG"
     fi
     if [[ $ARCH == "x86_64" && "$KERNEL_VERSION" == @(all|5.10-no-acpi) ]]; then
-        build_al_kernel $PWD/guest_configs/microvm-kernel-ci-$ARCH-5.10-no-acpi.config
+        build_al_kernel $PWD/guest_configs/microvm-kernel-ci-$ARCH-5.10-no-acpi.config "$CI_CONFIG"
     fi
     if [[ "$KERNEL_VERSION" == @(all|6.1) ]]; then
-        build_al_kernel $PWD/guest_configs/microvm-kernel-ci-$ARCH-6.1.config 5.10
+        build_al_kernel $PWD/guest_configs/microvm-kernel-ci-$ARCH-6.1.config "$CI_CONFIG"
     fi
 
-    # Undo kernel patches on top of AL configuration
-    git restore $PWD/guest_configs
-    rm -rf $PWD/guest_configs/*.orig 
+    # Build debug kernels
+    FTRACE_CONFIG="$PWD/guest_configs/ftrace.config"
+    DEBUG_CONFIG="$PWD/guest_configs/debug.config"
+    OUTPUT_DIR=$OUTPUT_DIR/debug
+    mkdir -pv $OUTPUT_DIR
+    build_al_kernel "$PWD/guest_configs/microvm-kernel-ci-$ARCH-5.10.config" "$CI_CONFIG" "$FTRACE_CONFIG" "$DEBUG_CONFIG"
+    vmlinux_split_debuginfo $OUTPUT_DIR/vmlinux-5.10.*
+    build_al_kernel "$PWD/guest_configs/microvm-kernel-ci-$ARCH-6.1.config" "$CI_CONFIG" "$FTRACE_CONFIG" "$DEBUG_CONFIG"
+    vmlinux_split_debuginfo $OUTPUT_DIR/vmlinux-6.1.*
```

Comment:
should we build these depending on the passed `$KERNEL_VERSION`?

Reply:
Ah true, I missed that.

Reply:
Fixed in latest revision